outputsheets/d20/5e/pdf/common_sheet/block_spells_condensed.xslt

```diff
@@ -30,10 +30,10 @@
 					<xsl:if test="warlock &gt; 0">
 						<xsl:apply-templates select="spell_slots_warlock"/>
 					</xsl:if>
-					<xsl:apply-templates select="spells_innate/racial_innate"/>	
-					<xsl:apply-templates select="spells_innate/class_innate"/>	
+<!--					<xsl:apply-templates select="spells_innate/racial_innate"/>			-->
```

Comment:
Is this a workaround for a code bug? Not showing racial innate spells seems odd

Reply:
Yes, it's a hack til we figure out why this is causing an NPE.